reqMgrClient.py

```diff
@@ -399,8 +427,8 @@ def getWorkflowInfo(url, workflow):
     """
```

Comment:
@ticoann you meant that the commands I posted above actually worked for you?
Because I couldn't make them work, this is my actual test script.
https://github.com/julianbadillo/WmAgentScripts/blob/jbadillo-work/test_reqmgrscripts.py